content/pascal/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,109 @@
+---
+Title: 'Data Types'
+Description: 'Pascal supports different data types that represent the size and kind of values being stored in memory.'
+Subjects:
+ - 'Computer Science'
+ - 'Game Development'
+Tags:
+ - 'Data Types'
+ - 'Integers'
+ - 'Booleans'
+ - 'Characters'
+ - 'Strings'
+CatalogContent:
+ - 'learn-pascal'
+ - 'paths/computer-science'
+---
+## Memory Size
+The size of a given data type is measured in bytes:
+
+| Data Type     | Memory Size |
+| ------------- | ----------- |
+| `boolean`     | 1 byte      |
+| `byte`        | 1 byte      |
+| `ShortInt`    | 2 bytes     |
+| `word`        | 2 bytes     |
+| `LongInt`     | 4 bytes     |
+| `Real`        | 4 bytes     |
+| `Double`      | 8 bytes     |
+
+
+## String	variable
+`Integers`
+Integer is a type for storing integer (whole) numbers. An integer usually requires 4 bytes of memory space and ranges from -2<sup>31</sup> to 2<sup>31</sup>-1.
```

Comment:
```suggestion
An `Integer` is a type for storing integer (whole) numbers. An integer usually requires 4 bytes of memory space and ranges from -2<sup>31</sup> to 2<sup>31</sup>-1.
```

Reply:
Explain the relationship between Integer and ShortInt and LongInt from the table above.